docs/dgComponent/AddRecipeCommand.md

```diff
@@ -0,0 +1,32 @@
+### Add recipe feature
+
+Add recipe feature allows users to add their personal recipes into the collection.
+
+#### Implementation
+Substitutability is used in Command and Parser:
+* `AddRecipeCommand` extends `Command`
+* `AddRecipeCommandParser` implements `Parser<AddRecipeCommand>`
+
+The following sequence diagram shows how add recipe operation works when `execute(addR n/Salad i/Veggies - 100g img/images/healthy3.jpg instr/Eat tag/healthy)` API call:
+
+![AddRecipeSequenceDiagram](../images/AddRecipeSequence.png)
+
+1. `LogicManager` receives user input and undergoes logic operation for output.
+1. `LogicManager` will pass the input to `WishfulShrinkingParser`.
+1. `WishfulShrinkingParser` creates `AddRecipeCommandParser` to parse and validate the user input.
+1. `AddRecipeCommandParser` creates `AddRecipeCommand` with successfully parsed input.
+1. `LogicManager` executes `AddRecipeCommand`.
+1. Pass the `Recipe` to `Model` which responsible in adding it to `UniqueRecipeList`.
+1. `AddRecipeCommand` return a `CommandResult` back to `LogicManager`.
```

Comment:
Might want to consider changing this to the explanation like delete consumption and delete ingredients